src/data/queries/searchDoiQuery.ts

```
@@ -10,17 +10,18 @@ function extractRORId(rorString: string): string {
   return rorString.replace('https://', '').replace('ror.org/', '')
 }
 
-function buildOrgQuery(rorId: string | undefined): string {
+function buildOrgQuery(rorId: string | undefined, rorFundingIds: string[]): string {
   if (!rorId) return ''
   const id = 'ror.org/' + extractRORId(rorId)
   const urlId = `"https://${id}"`
-  return `(organization_id:${id} OR affiliation_id:${id} OR related_dmp_organization_id:${id} OR provider.ror_id:${urlId})`
+  const rorFundingIdsQuery = rorFundingIds.map(id => '\"https://doi.org/' + id + '\"').join(' OR ')
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)